src/test/scala/com/cognite/sdk/scala/common/SdkTest.scala

```diff
@@ -30,7 +30,11 @@ class LoggingSttpBackend[R[_], S](delegate: SttpBackend[R, S]) extends SttpBacke
 
 abstract class SdkTest extends FlatSpec with Matchers {
   // Use this if you need request logs for debugging: new LoggingSttpBackend[Id, Nothing](sttpBackend)
-  val client = new GenericClient("scala-sdk-test")(implicitly[Monad[Id]], auth, sttpBackend)
+  val client = new GenericClient("scala-sdk-test", "api.cognitedata.com")(
+    implicitly[Monad[Id]],
+    auth,
+    sttpBackend
```

Comment:
I like the formatting change here but we should move the comment to above this line:
```scala

    // Use this if...
    sttpBackend
```